ui-automation-tests/acceptance-features/services/add-service.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Add a new service
+
+  Background: User navigating to /services page
+    Given the user is on the "/sign-in" page
+    And they submit the email "registered@test.gov.uk"
+    And they submit a valid password
+    And they submit a correct security code
+    Then they should be redirected to a page with the title "Your services - GOV.UK One Login"
+
+  Rule: The user validating add a service option in services page
```

Comment:
This Rule and step is not needed as the existence of the link is already validated when you click on it in the next step in line 17